src/guest/tui.cc

```diff
@@ -176,8 +186,22 @@ void CivTui::InitializeForm(void) {
 
 void CivTui::InitializeButtons(void) {
     SaveOn = [&]() {
-        screen_.ExitLoopClosure();
-        status_bar_ += "called SaveOn!";
+        std::string configPath = GetConfigPath();
+        std::string filePath = configPath + "/" + filename_ +".ini";
+        if (!boost::filesystem::exists(filePath)) {
+            boost::filesystem::ofstream File(filePath) ;
```

Comment:
local variable starts with lower case. File -> file.
BTW, seems the File is just open and closed, no other operation on it. can you remove this variable?